spec/features/users_spec.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.feature 'Users', type: :feature do
+  context 'registrations users' do
+    scenario 'empty email and password' do
```

Comment:
All descriptions should be clear:

'when user registers with empty email and password' do

or 

```
context 'when user registers ...' do
  context  '... with empty email and password' do
    ...
    it 'shows an error' do
    ...
  
```